<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>285</width>
      <height>349</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="ecran_pile"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="ecran">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QPushButton" name="button_0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="shortcut">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="button_9">
         <property name="text">
          <string>9</string>
         </property>
         <property name="shortcut">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_8">
         <property name="text">
          <string>8</string>
         </property>
         <property name="shortcut">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="button_1">
         <property name="text">
          <string>1</string>
         </property>
         <property name="shortcut">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="button_4">
         <property name="text">
          <string>4</string>
         </property>
         <property name="shortcut">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_7">
         <property name="text">
          <string>7</string>
         </property>
         <property name="shortcut">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="button_5">
         <property name="text">
          <string>5</string>
         </property>
         <property name="shortcut">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="button_2">
         <property name="text">
          <string>2</string>
         </property>
         <property name="shortcut">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="button_6">
         <property name="text">
          <string>6</string>
         </property>
         <property name="shortcut">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="button_3">
         <property name="text">
          <string>3</string>
         </property>
         <property name="shortcut">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QPushButton" name="button_egal">
         <property name="text">
          <string>=</string>
         </property>
         <property name="shortcut">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="button_plus">
         <property name="text">
          <string>+</string>
         </property>
         <property name="shortcut">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="button_fois">
         <property name="text">
          <string>*</string>
         </property>
         <property name="shortcut">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="button_divise">
         <property name="text">
          <string>/</string>
         </property>
         <property name="shortcut">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="button_moins">
         <property name="text">
          <string>-</string>
         </property>
         <property name="shortcut">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QPushButton" name="button_space">
         <property name="text">
          <string>SPACE</string>
         </property>
         <property name="shortcut">
          <string>Space</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="button_clear">
         <property name="text">
          <string>CLEAR</string>
         </property>
         <property name="shortcut">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>305</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
